demo/routes/index.svelte

```diff
@@ -231,6 +231,18 @@
       <code class="code">border-radius</code> through the custom properties shown.
     </p>
     <p class="mb-5">This is also compatible with custom languages.</p>
+    <p class="mb-5">
+      It is recommended that you set values for
+      <code class="code">--hljs-background</code> and
+      <code class="code">--hljs-foreground</code> to ensure the langtags remain readable
+      on any theme.
+    </p>
+    <p class="mb-5">Defaults:</p>
+    <ul class="mb-5">
+      <li><code class="code">--hljs-background: inherit</code></li>
+      <li><code class="code">--hljs-foreground: inherit</code></li>
+      <li><code class="code">--hljs-border-radius: 0</code></li>
+    </ul>
```

Comment:
Let's use the list components from Carbon which have the correct line-heights/padding.

```suggestion
    <UnorderedList class="mb-5">
      <ListItem><code class="code">--hljs-background: inherit</code></ListItem>
      <ListItem><code class="code">--hljs-foreground: inherit</code></ListItem>
      <ListItem><code class="code">--hljs-border-radius: 0</code></ListItem>
    </UnorderedList>
```

Reply:
Good call, this will keep styles consistent.